src/patient-registration/section/relationship-section.component.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react';
+import styles from './section.css';
+import { RelationshipInput } from '../input/custom-input/relationship-input.component ';
+
+interface RelationshipProps {
+  setFieldValue(field: string, value: any, shouldValidate?: boolean): void;
+}
+
+export const RelationshipSection: React.FC<RelationshipProps> = ({ setFieldValue }) => {
+  return (
+    <section className={styles.formSection}>
+      <h2 className="omrs-type-title-2">Relationships</h2>
```

Comment:
There's duplicate _Relationships_ headers here. The second one can be removed.

Reply:
Done